### PR TITLE
DB-11551 make KillOperationIT more robust

### PR DESCRIPTION
**Pull request by martinrupp**
## Description
instead of waiting exactly one second, then check SYSCS_GET_RUNNING_OPERATIONS for the operation that was submitted, we now wait 100 x 200 ms and check, which should increase robustness against situations where the submission of the task takes a bit longer.

### CODE REVIEW

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Review by jyuanca — APPROVED** [review]

**Comment by cloudspliceci** [comment]
TEST SUCCEEDED +1

File: hbase_sql/src/test/java/com/splicemachine/hbase/KillOperationIT.java [hunk 1]
```diff
@@ -208,6 +198,38 @@ public void run() {
...
+    private String executeRunningOperations(String sql, TestConnection connection) throws SQLException, InterruptedException {
+
+        int count = 0;
+        String uuid = null;
+        // 2 running operations, cursor + the procedure call itself
+        for(int i=0; i<100 && count != 2; i++) {
+            // wait for the query to be submitted
+            Thread.sleep(200);
+
+            String opsCall = "call SYSCS_UTIL.SYSCS_GET_RUNNING_OPERATIONS()";
```
Comment (hatyo) [inline hbase_sql/src/test/java/com/splicemachine/hbase/KillOperationIT.java:216]: could also be `final`.

**Review by hatyo — APPROVED** [review]
Nice cleanup, good work!

**Review by carolp-503 — APPROVED** [review]
+1

**Pull request merged**